modules/context/org.go

```diff
@@ -17,10 +17,11 @@ import (
 
 // Organization contains organization context
 type Organization struct {
-	IsOwner          bool
-	IsMember         bool
-	IsTeamMember     bool // Is member of team.
-	IsTeamAdmin      bool // In owner team or team that has admin permission level.
+	IsOwner          bool // Is owner of the org
+	IsMember         bool // Is member of the org
+	IsAdmin          bool // Is admin of the org
+	IsTeamMember     bool // Is member of team, only used in urls which has team param
+	IsTeamAdmin      bool // Is in owner team or team that has admin permission level, only used in urls which has team param
```

Comment:
What's the different between `IsTeamAdmin` and `IsAdmin`?

Reply:
:raised_eyebrow: Sounds like they are the same.
But if that is indeed the case, I still support renaming `IsTeamAdmin` to `IsAdmin` or `IsOrgAdmin` (if we are concerned about confusion with site admins) as `IsTeamAdmin` reads as if the user only has admin rights within the team instead of the whole org.

Reply:
First, `ctx.Org.IsTeamAdmin` is only used in `OrgAssignment` function for checking `requireTeamAdmin`.
![image](https://github.com/go-gitea/gitea/assets/18380374/eec7e32e-8b56-4d90-a563-9dc94ba7cd4d)

Then, `ctx.Data["IsTeamAdmin"]` is empty when the URL doesn't have `team` param.
So if we want to detect whether doer is the admin in these pages.
So I think we need `ctx.Data["IsOrganizationAdmin"]`, and `ctx.Org.IsTeamAdmin` can be removed here.